pkg/cache/cache_test.go

```diff
@@ -1544,7 +1544,7 @@ func CacheTest(createCacheFunc func(config *rest.Config, opts cache.Options) (ca
 					return obtainedPodNames
 				}, ConsistOf(tc.expectedPods)))
 				for _, pod := range obtainedStructuredPodList.Items {
-					Expect(informer.Get(context.Background(), client.ObjectKeyFromObject(&pod), &pod)).To(Succeed()) //nolint:gosec // We don't retain the pointer
+					Expect(informer.Get(context.Background(), client.ObjectKeyFromObject(&pod), &pod)).To(Succeed()) // We don't retain the pointer
```

Comment:
Can delete the entire comments for these, the purpose of the comment was to explain the `nolint` directive